src/coverlet.core/Instrumentation/ReachabilityHelper.cs

```diff
@@ -224,7 +226,9 @@ public override string ToString()
         /// OpCodes that unconditionally transfer control, so there
         /// is not "fall through" branch target.
         /// </summary>
+#pragma warning disable IDE1006 // Naming Styles
```

Comment:
I thought you would prefer not to change the name of the field but if you are ok with changing, please let me know.

Reply:
I'm ok with it